components/image_generation/spaceFiller.tsx

```diff
@@ -3,74 +3,76 @@ import RND from "./randomizer";
 const canvasWidth = 478;
 const canvasHeight = 478;
 
-const c = 1.1 
+const c = 1.1;
```

Comment:
haha this file was leftover from another project - it tries to do randomly place objects without them overlapping - i can remove it